react-validated/package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
@ardianck what is this file for?

Reply:
@visar-polymath this is generated by package.json. Maybe I did not get your question?

Reply:
@ardianck note the path, it was a bad artifact... I removed that whole folder